tests/operators/test_constraints.py

```diff
@@ -33,6 +33,15 @@ def test_generate_var_constraint():
     assert repr(var_constraint) == expected_var_constraint
 
 
+def test_generate_model_level_constraint():
+    """Generate iris cube constraint for str variable name."""
+    var_constraint = constraints.generate_model_level_constraint("2")
+    expected_model_level_constraint = (
+        "Constraint(coord_values={'model_level_number': 2})"
```

Comment:
```suggestion
        "Constraint(coord_values={'model_level_number': '2'})"
```
The model level number comes out the constraint as a string.